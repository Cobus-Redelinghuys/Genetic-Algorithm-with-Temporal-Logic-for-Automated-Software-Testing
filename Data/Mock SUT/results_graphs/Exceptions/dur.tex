Table \ref{tab:HP:GA:Exceptions:duration} tabulates the final average, standard deviations, and overall average for the duration that the different GATLAST configurations configured with different exceptions weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
exceptions & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 40353.6 & 2384.211 & 41870.53\\\hline
0.25 & 40968.6 & 2616.726 & 41790.912\\\hline
0.5 & 40847.0 & 3322.273 & 41340.126\\\hline
0.75 & 41302.8 & 2818.863 & 41877.258\\\hline
1.0 & 43591.0 & 5953.926 & 41559.514\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the duration metric that the of different exceptions configured GATLASTs were achieved}
\label{tab:HP:GA:Exceptions:duration}
\end{table}