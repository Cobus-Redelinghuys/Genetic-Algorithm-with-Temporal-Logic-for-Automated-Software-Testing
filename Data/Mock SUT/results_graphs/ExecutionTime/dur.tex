Table \ref{tab:HP:GA:ExecutionTime:duration} tabulates the final average, standard deviations, and overall average for the duration that the different GATLAST configurations configured with different execution time weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
execution time & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 41848.3 & 3311.059 & 41996.67\\\hline
0.25 & 42099.2 & 4466.905 & 41858.452\\\hline
0.5 & 40847.0 & 3322.273 & 41340.126\\\hline
0.75 & 40801.3 & 3582.191 & 41757.148\\\hline
1.0 & 41863.2 & 3434.306 & 41672.446\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the duration metric that the of different execution time configured GATLASTs were achieved}
\label{tab:HP:GA:ExecutionTime:duration}
\end{table}