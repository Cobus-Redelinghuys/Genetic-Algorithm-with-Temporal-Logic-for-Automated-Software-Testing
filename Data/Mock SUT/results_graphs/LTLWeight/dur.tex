Table \ref{tab:HP:GA:LTLWeight:duration} tabulates the final average, standard deviations, and overall average for the duration that the different GATLAST configurations configured with different ω₁ weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
ω₁ & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 39899.9 & 2264.392 & 41046.46\\\hline
0.25 & 40939.6 & 3104.631 & 41666.732\\\hline
0.5 & 40847.0 & 3322.273 & 41340.126\\\hline
0.75 & 45353.8 & 5342.161 & 41927.932\\\hline
1.0 & 40497.4 & 1902.093 & 41729.588\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the duration metric that the of different ω₁ configured GATLASTs were achieved}
\label{tab:HP:GA:LTLWeight:duration}
\end{table}