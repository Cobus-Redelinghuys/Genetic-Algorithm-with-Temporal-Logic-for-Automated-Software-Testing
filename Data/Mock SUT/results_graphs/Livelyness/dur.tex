Table \ref{tab:HP:GA:Livelyness:duration} tabulates the final average, standard deviations, and overall average for the duration that the different GATLAST configurations configured with different proper termination weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
proper termination & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 39630.0 & 470.484 & 41319.578\\\hline
0.25 & 42527.4 & 4770.009 & 41818.666\\\hline
0.5 & 40847.0 & 3322.273 & 41340.126\\\hline
0.75 & 41208.1 & 4430.213 & 41615.142\\\hline
1.0 & 41920.3 & 4559.836 & 41601.256\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the duration metric that the of different proper termination configured GATLASTs were achieved}
\label{tab:HP:GA:Livelyness:duration}
\end{table}