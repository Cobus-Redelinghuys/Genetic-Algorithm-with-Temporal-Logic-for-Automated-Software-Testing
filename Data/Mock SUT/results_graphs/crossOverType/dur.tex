Table \ref{tab:HP:GA:crossOverType:duration} tabulates the final average, standard deviations, and overall average for the duration that the different GATLAST configurations configured with different crossover type weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
crossover type & Final average & Final standard deviation & Overall average\\
\hline
N-point crossover & 43036.1 & 4723.031 & 41934.15\\\hline
One point crossover & 40847.0 & 3322.273 & 41340.126\\\hline
Segmented crossover & 43941.3 & 8419.346 & 41507.84\\\hline
Uniform crossover & 41843.2 & 2683.698 & 42226.972\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the duration metric that the of different crossover type configured GATLASTs were achieved}
\label{tab:HP:GA:crossOverType:duration}
\end{table}