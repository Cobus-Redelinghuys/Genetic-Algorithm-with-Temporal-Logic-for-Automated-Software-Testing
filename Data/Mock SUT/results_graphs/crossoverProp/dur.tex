Table \ref{tab:HP:GA:crossoverProp:duration} tabulates the final average, standard deviations, and overall average for the duration that the different GATLAST configurations configured with different crossover probability weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
crossover probability & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 40485.0 & 2158.383 & 40683.73\\\hline
0.25 & 40965.9 & 3959.145 & 41248.792\\\hline
0.5 & 40847.0 & 3322.273 & 41340.126\\\hline
0.75 & 41938.8 & 6286.523 & 41863.038\\\hline
1.0 & 45683.3 & 8112.182 & 41817.788\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the duration metric that the of different crossover probability configured GATLASTs were achieved}
\label{tab:HP:GA:crossoverProp:duration}
\end{table}