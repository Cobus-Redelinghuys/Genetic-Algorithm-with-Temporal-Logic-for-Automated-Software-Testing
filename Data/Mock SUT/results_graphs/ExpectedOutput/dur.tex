Table \ref{tab:HP:GA:ExpectedOutput:duration} tabulates the final average, standard deviations, and overall average for the duration that the different GATLAST configurations configured with different expected output weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
expected output & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 40992.7 & 4430.189 & 41996.23\\\hline
0.25 & 40480.0 & 2348.735 & 41408.676\\\hline
0.5 & 40847.0 & 3322.273 & 41340.126\\\hline
0.75 & 42477.1 & 4606.227 & 41769.15\\\hline
1.0 & 43754.4 & 6336.055 & 41753.436\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the duration metric that the of different expected output configured GATLASTs were achieved}
\label{tab:HP:GA:ExpectedOutput:duration}
\end{table}